src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -383,7 +393,7 @@ private static object SanitizeSecurityArtifact(object arg)
                 return "null";
 
             if (IdentityModelEventSource.LogCompleteSecurityArtifact && arg is ISafeLogSecurityArtifact)
-                return (arg as ISafeLogSecurityArtifact).UnsafeToString();
+                return ((ISafeLogSecurityArtifact)arg).UnsafeToString();
```

Comment:
why this change?

Reply:
suggestion: use pattern matching to clean this up